src/NServiceBus.Core/Licensing/LicenseExpiredForm.cs

```diff
@@ -23,21 +24,27 @@ protected override void OnLoad(EventArgs e)
         {
             base.OnLoad(e);
 
-            warningText.Text = "The trial period is now over";
+            var version = Assembly.GetExecutingAssembly().GetName().Version;
 
             if (CurrentLicense != null && !CurrentLicense.IsExtendedTrial)
             {
-                Text = "NServiceBus - Initial Trial Expired";
-                instructionsText.Text = "To extend the free trial, click 'Extend trial' and register online. When the license file is received, save it to disk and then click the 'Browse' button below to select it.";
+                Text = $"NServiceBus License (v{version.ToString(3)})";
+                warningText.Text = "It's time to extend your trial.";
+                instructionsText.Text = @"Your 14-day trial is up, but you can instantly extend your trial for 30 days.
```

Comment:
What about
> Your 14-day trial is up, but you can instantly extend your trial for free.

I don't think we should put magic numbers in here especially is we can control extended evaluation period in the license file generation.